schema/url/standalone-map/plugin-floating-lineplot.raml

```diff
@@ -0,0 +1,55 @@
+#%RAML 1.0 Library
+
+types:
+  # Type name determines the name of the Java class that will be generated.
+
+  FloatingLineplotPostRequest:
+    type: VisualizationRequestBase
+    properties:
+      config: FloatingLineplotSpec
+
+  FloatingLineplotSpec:
+    additionalProperties: false
+    properties:
+      outputEntityId:
+        type: string
+      xAxisVariable: 
+        type: VariableSpec
+      yAxisVariable: 
+        type: VariableSpec
+      yAxisNumeratorValues:
```

Comment:
There's potentially another UI real estate challenge with the extended lineplot variables UI, but I think for now we keep this in and make space for it somehow.

Reply:
Thanks, but I think the custom fractions for categoricals are probably going to be quite useful. At least I can't think of a reason to get rid of them at this stage. We'll find some screen space somehow.

Reply:
for the sam, could we break up line into two in order to simplify plot config? One for categorical and one for continuous?

Reply:
That would be an option - let's see how it goes with what we have.  I think I am about to do a major lineplot refactor... :grimacing: